pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/NamespaceService.java

```diff
@@ -571,9 +571,7 @@ public boolean isNamespaceBundleDisabled(NamespaceBundle bundle) throws Exceptio
                 updateNamespaceBundles(nsname, splittedBundles.getLeft(),
                         (rc, path, zkCtx, stat) -> pulsar.getOrderedExecutor().submit(safeRun(() -> {
                             if (rc == KeeperException.Code.OK.intValue()) {
-                                // disable old bundle
                                 try {
-                                    ownershipCache.disableOwnership(bundle);
```

Comment:
Is this needed, or just a safety measure to have the parent bundle to be still "active" in case other brokers don't have the local cache updated yet?

Reply:
yes, that protection may need while broker is refreshing cache (invalidate old bundle and assign topic-ownership to new bundles). So, I will add that protection again by disabling namespace in memory only but other broker can still redirect to this broker.